wiki/Ranking_criteria/osu!mania/en.md

```diff
@@ -19,8 +19,13 @@ This set of **osu!mania ranking criteria** lays out [rules and guidelines](/wiki
 ### Common terms
 
 - **BPM**: Tempo of a song measured in beats per minute.
-- **Column**: Vertical lines on the playfield where notes are placed. The editor uses `0|1|2|3|4|5|6|7|8|9` for timestamp labels.
+- **Column**: Vertical slices of the [playfield](/wiki/Game_mode/osu!mania#playfield) where notes are placed. Note that [editor timestamps](/wiki/Modding/Timestamp) refer to columns by their [zero-indexed](https://en.wikipedia.org/wiki/Zero-based_numbering "\"Zero-based numbering\" on Wikipedia") position starting from the left.
 - **Key mode**: The number of columns present in a difficulty.
+- **Playstyle**: A mapper's intended style of [key bindings](/wiki/Game_mode/osu!mania#controls) or hand positions used to play a difficulty.
+- **Standard playstyle**: The most common type of playstyle for key counts 10 or below, where no column is of particular significance, and the player is expected to control each key with a unique finger.
+  - Examples of standard playstyles: 4K, 7K, 10K
+  - Examples of non-standard playstyles: 7K+1, 10K2S, 14K DP
+- **N+1 playstyle**: A specific type of playstyle where the leftmost column is mapped independently from the rest of the columns, which otherwise form a standard playstyle. Playstyles may also be considered "N+2" if they add a second independently-mapped column.
```

Comment:
from the perspective of the mapper it always goes on left, then the player chooses if it goes on the left or right. this definition and the rule that refers to it are only really here to help explain conventional usage of the beatmap setting, which is also only used to mark a special column on the left. it sounds weird because it is weird...

Reply:
Since beginning, osu!mania stable editor always treats leftmost column as special column on N+1 maps, despite honoring mapper's skin preference for coloring purposes.